.github/workflows/airbyte-ci-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Airbyte CI pipeline tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - airbyte-ci/*
+jobs:
+  run-airbyte-ci-tests:
+    name: Run Airbyte CI tests
+    runs-on: "conn-prod-xlarge-runner"
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+      - name: Run pipelines tests
+        id: run-pipelines-tests
+        uses: ./.github/actions/run-dagger-pipeline
+        with:
+          context: "pull_request"
+          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
+          gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
+          github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
+          subcommand: "tests connectors/pipelines"
```

Comment:
I really like this pattern. 

Can we update the metadata service test to use this?
1. https://github.com/airbytehq/airbyte/blob/master/.github/workflows/metadata_service_tests_dagger.yml#L31
2. https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/commands/groups/metadata.py#L120-L154

Let me know if out of scope

Reply:
@bnchrch as this new workflow is not on master I'd prefer expanding its scope in differents PRs so that I can check it's working correctly 😄

Reply:
Great! Would you mind creating a small issue for these then. Would love to update them :)

Reply:
Will do